app/src/main/java/mozilla/org/webmaker/WebmakerApplication.java

```diff
@@ -32,7 +32,8 @@ public void onCreate() {
         Router.sharedRouter().map("/main/:tab", MainActivity.class);
         Router.sharedRouter().map("/map/:projectId", Map.class);
         Router.sharedRouter().map("/projects/:projectId", Project.class);
-        Router.sharedRouter().map("/projects/:projectId/elements/:elementId", Editor.class);
+        Router.sharedRouter().map("/edit/:projectId/:elementId/:type", Editor.class);
```

Comment:
not really: the routing is necessary to have the correct element pull up the correct editor on Android, which cannot hook into window.location.has like editor.jsx does right now (That only works in the desktop browser)